variants/heltec_vision_master_e213/variant.h

```diff
@@ -1,8 +1,13 @@
+#ifndef VARIANT_H
+#define VARIANT_H
 // #define LED_PIN 18
 
+#ifdef HELTEC_VISION_MASTER_E213
+#define HELTEC_VISION_MASTER_E213 true
+
 // Enable bus for external periherals
-#define I2C_SDA SDA
-#define I2C_SCL SCL
+#define I2C_SCL 38
+#define I2C_SDA 39
```

Comment:
Were these working when defined as `SDA` and `SCL`, once you fixed pins_arduino.h?

Reply:
Ill test it

Reply:
yeah it works after I changed this back to sda